internal/function_time_parser.go

```diff
@@ -1091,6 +1091,29 @@ func parseTimeFormat(formatStr, targetStr string, typ TimeFormatType) (*time.Tim
 				return nil, fmt.Errorf("invalid time format")
 			}
 			c = format[formatIdx]
+			if c == 'E' {
```

Comment:
Could you please add this difference to `formatPatternMap` like 'e' case ?

```
'E': &FormatTimeInfo{
	AvailableTypes: []TimeFormatType{
		FormatTypeDate, FormatTypeDatetime, FormatTypeTimestamp,
	},
	Parse:  // specify the parser
	Format: // specify the formatter
}
```

Reply:
@goccy 
Thank you for reviewing.

I got the point.
But I'm concerned it needs to parse multi characters like 'Ez', 'E*S' or 'E4Y'.
So, I think `formatPatternMap` should be extended to handle multi characters like: 
```
var formatPatternMap = map[string]*FormatTimeInfo{
   "Ez": &FormatTimeInfo{
        AvailableTypes: []TimeFormatType{
            FormatTypeDate, FormatTypeDatetime, FormatTypeTimestamp,
        },
        Parse:  // specify the parser
        Format: // specify the formatter
    },
   "E*S": ...
}
```

And we also need to get format string by looking some step ahead instead of simply `c := format[formatIdx]`.
```
p, formatProgress, err := getFormatPattern(format[formatIdx:])
```

Is it acceptable change?

Reply:
I see. This changes looks good ! Thank you !